File: parse_naver.js
import * as cheerio from 'cheerio';
import * as fs from 'fs';
async function run() {
  const html = fs.readFileSync('naver.html', 'utf-8');
  const $ = cheerio.load(html);
  $('.api_txt_lines.total_tit').each((i, el) => {
    console.log($(el).text());
  });
}
run();

File: parse_naver.py
from bs4 import BeautifulSoup
import sys

with open("naver.html", "r", encoding="utf-8") as f:
    html = f.read()

soup = BeautifulSoup(html, "html.parser")
for a in soup.select("a.api_txt_lines.total_tit"):
    print(a.text)
